Pod/Classes/SeafConnection.m

```diff
@@ -488,6 +493,13 @@ - (NSString *)uniqueUploadDir
     return [SeafStorage uniqueDirUnder:self.localUploadDir];
 }
 
+- (NSString *)tagDataKey {
+    if (!_tagDataKey) {
+        _tagDataKey = [NSString stringWithFormat:@"%@%@",TAGDATA,self.username];
```

Comment:
use accountIdentifier instead of username

Reply:
accountIdentifier is too long

Reply:
but username is not unique, two accounts can have the same username